src/inter-aut.cc

```diff
@@ -308,7 +309,7 @@ namespace {
      * @param postfix Postfix to which operators are eventually added
      * @return A postfix with eventually added operators
      */
-    std::vector<mata::FormulaNode> add_disjunction_implicitly(const std::vector<mata::FormulaNode> &postfix)
+    std::vector<mata::FormulaNode> add_disjunction_implicitly(std::vector<mata::FormulaNode> postfix)
```

Comment:
Why did you remove the `&` reference? Isn't this less efficient?

Reply:
Because we are moving the data from the caller to the function parameter. Hence, we can afterwards move the elements of the postfix vector into the output vector. Otherwise, we would have to create a new copy in the output vector.